src/core/zowe/core_for_zowe_sdk/sdk_api.py

```diff
@@ -56,6 +56,12 @@ def __init__(self, profile, default_url, logger_name = __name__):
             self.default_headers["Authorization"] = f"Bearer {self.session.tokenValue}"
         elif self.session.type == session_constants.AUTH_TYPE_TOKEN:
             self.default_headers["Cookie"] = f"{self.session.tokenType}={self.session.tokenValue}"
+    
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exception, traceback):
```

Comment:
Will there be any use of `exc_type, exception, traceback`? 
I am curious :thinking: